impeller/entity/contents/runtime_effect_contents.h

```diff
@@ -35,7 +35,16 @@ class RuntimeEffectContents final : public ColorSourceContents {
               const Entity& entity,
               RenderPass& pass) const override;
 
+  /// Load the runtime effect and ensure a default PSO is initialized.
+  bool BootstrapShader(const ContentContext& renderer) const;
```

Comment:
I might need to move these utilities around to make the imports legal, but it basically works